package com.gitz.dicodingevent.data.local.entity

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "favorite_event")
data class FavoriteEvent(
    @PrimaryKey(autoGenerate = false)
    var id: Int = 0,
    var name: String = "",
    var mediaCover: String? = null,
)